v2/concordium/types.proto

```diff
@@ -2037,6 +2037,14 @@ message UpdatePayload {
   }
 }
 
+// Details about the sponsor of a transaction.
+message SponsorDetails {
+  // The cost of the transaction. Paid by the sponsor.
+  Amount cost = 1;
+  // The sponsor of the transaction.
+  AccountAddress sponsor = 2;
+}
+
 // Details about an account transaction.
 message AccountTransactionDetails {
   // The cost of the transaction. Paid by the sender.
```

Comment:
I would add something to the effect that if the transaction is sponsored then this will be 0.